src/VisualStudio/Core/Def/TableDataSource/Suppression/VisualStudioSuppressionFixService.cs

```diff
@@ -277,6 +285,9 @@ private async Task ApplySuppressionFixAsync(IEnumerable<DiagnosticData>? diagnos
 
                 var cancellationToken = context.UserCancellationToken;
                 cancellationToken.ThrowIfCancellationRequested();
+
+                var progress = context.GetCodeAnalysisProgress();
```

Comment:
>var progress = context.GetCodeAnalysisProgress(); [](http://example.com/codeflow?start=16&length=49)

nit: defer until after the "Nothing to fix" check

Reply:
done!